src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/BookingFacade.kt

```diff
@@ -141,4 +146,6 @@ class BookingFacade(
   private fun Collection<PrisonAppointment>.main() = single { it.appointmentType == "VLB_COURT_MAIN" }
 
   private fun Collection<PrisonAppointment>.post() = singleOrNull { it.appointmentType == "VLB_COURT_POST" }
+
+  private fun Map<String, Location>.room(key: String) = this[key]?.localName ?: ""
```

Comment:
Question: If no name leave empty or put some default?

Technically this will be the same as whatever room they chose so should not be empty.